agent/consul/leader.go

```diff
@@ -1102,6 +1107,9 @@ AFTER_CHECK:
 			Output:  structs.SerfCheckAliveOutput,
 		},
 		EnterpriseMeta: *nodeEntMeta,
+		NodeMeta: map[string]string{
+			structs.MetaConsulVersion: (*buildVersion).String(),
```

Comment:
should `buildVersion.String()` work as well?

Reply:
Yes updated